packages/contracts-bedrock/test/LivenessModule.t.sol

```diff
@@ -0,0 +1,82 @@
+// SPDX-License-Identifier: MIT
+pragma solidity 0.8.15;
+
+import { Test, StdUtils } from "forge-std/Test.sol";
+import { Safe } from "safe-contracts/Safe.sol";
+import { SafeProxyFactory } from "safe-contracts/proxies/SafeProxyFactory.sol";
+import { ModuleManager } from "safe-contracts/base/ModuleManager.sol";
+import { Enum } from "safe-contracts/common/Enum.sol";
+import "test/safe-tools/SafeTestTools.sol";
+
+import { LivenessModule } from "src/Safe/LivenessModule.sol";
+import { LivenessGuard } from "src/Safe/LivenessGuard.sol";
+
+contract LivnessModule_TestInit is Test, SafeTestTools {
+    using SafeTestLib for SafeInstance;
+
+    event SignersRecorded(bytes32 indexed txHash, address[] signers);
+
+    LivenessModule livenessModule;
+    LivenessGuard livenessGuard;
+    SafeInstance safeInstance;
+
+    function setUp() public {
+        // Create a Safe with 10 owners
+        (, uint256[] memory keys) = makeAddrsAndKeys(10);
+        safeInstance = _setupSafe(keys, 8);
+        livenessGuard = new LivenessGuard(safeInstance.safe);
+        livenessModule = new LivenessModule({
+            _safe: safeInstance.safe,
+            _livenessGuard: livenessGuard,
+            _livenessInterval: 30 days,
+            _minOwners: 6,
+            _fallbackOwner: makeAddr("fallbackOwner")
+        });
+        safeInstance.enableModule(address(livenessModule));
+        safeInstance.setGuard(address(livenessGuard));
+    }
+}
+
+contract LivenessModule_Get75PercentThreshold_Test is LivnessModule_TestInit {
+    /// @dev check the return values of the get75PercentThreshold function against manually
+    ///      calculated values.
+    function test_get75PercentThreshold_Works() external {
+        assertEq(livenessModule.get75PercentThreshold(20), 15);
```

Comment:
I have this intuition that we want roundtrip testing to ensure there are not any precison errors because of `_verifyFinalState`. Do you think this makes sense?

Reply:
Perhaps, let's agree on a spec for that test, how does this sound: 

1. initialize a safe with a random number of signers (up to 20).
2. add a random number of signers
3. remove the same number of signers (without going below the minOwners threshold)
4. assert that the threshold is unchanged. 

Another test could do the same but removing before adding. 
An even better test would run a randomized series of additions/removals.

Reply:
These sound good to me

Reply:
This is tracked in the PR description checklist. Resolving.